MainForm.cs

```diff
@@ -98,6 +100,9 @@ public void FindHighCard(ref int highCard, ref int higherCardNumber, int combLen
                     highCard = nomOfCards[higherCardNumber];
                 }
             }
+            arr[0] = highCard;
```

Comment:
It is considered good practice to use the new class as parameters storage here. But no changes are needed now, we will implement this logic with the next task.